plugins/nf-validation/src/main/nextflow/validation/SamplesheetConverter.groovy

```diff
@@ -64,15 +60,13 @@ class SamplesheetConverter {
         JSONObject rawSchema = new JSONObject(new JSONTokener(schemaFile.text))
         SchemaLoader schemaLoader = SchemaLoader.builder()
                 .schemaJson(rawSchema)
-                .addFormatValidator("file-path", new FilePathValidator())
-                .addFormatValidator("directory-path", new DirectoryPathValidator())
```

Comment:
Where are these checks now? I can't find them

Reply:
They are in the function to validate files https://github.com/mirpedrol/nf-validation/blob/automatic-validation/plugins/nf-validation/src/main/nextflow/validation/SchemaValidator.groovy#L376
I removed it to avoid checking the same thing twice. But now that I think about, maybe it's better to validate too much than too little, I will add this back.

Reply:
Ah okay I missed that! It's okay if it's only there (that code is also used for the samplesheet now, right?)

Reply:
It's also used for samplesheets yes. But could we have the case where someone wants to convert a file to a channel and doesn't validate it? It's not ideal, but I don't know if it will happen

Reply:
Ah yes I see what you mean hmmm. Maybe we can add some kind of check, if possible, to see if the samplesheet has been validated before converting it? So that it throws a warning to the user if they didn't validate their samplesheet?

Reply:
I was trying to use `validateAndConvertSamplesheet()` obtaining the path from the parameters JSON schema (suggested by Phil) instead of typing the path in the pipeline. This would be ideal I think, because then it enforces you to have it specified in the schema, and it will always be validated together with parameters.

Reply:
Even better! That should definitely do it :)